src/DAL/repositories/heartbeatRepository.ts

```diff
@@ -26,6 +26,13 @@ const createHeartbeatRepository = (dataSource: DataSource) => {
       const deleted = await this.remove(entities);
       return deleted.length;
     },
+
+    async getHeartbeat(_id: string): Promise<HeartbeatEntity | null> {
+      const entity = await this.findOneBy({
+        id: Equal(_id),
```

Comment:
what is the added value of the `Equal` function?
can work the same this way as well:
![image](https://github.com/user-attachments/assets/06da6c78-0c68-4b62-a632-82c93a78ecf8)